libcontainer/intelrdt/intelrdt.go

```diff
@@ -21,19 +21,26 @@ import (
 /*
  * About Intel RDT features:
  * Intel platforms with new Xeon CPU support Resource Director Technology (RDT).
- * Cache Allocation Technology (CAT) and Memory Bandwidth Allocation (MBA) are
- * two sub-features of RDT.
+ * Cache Allocation Technology (CAT), Cache Monitoring Technology (CMT),
+ * Memory Bandwidth Allocation (MBA) and Memory Bandwidth Monitoring (MBM) are
+ * four sub-features of RDT.
  *
  * Cache Allocation Technology (CAT) provides a way for the software to restrict
  * cache allocation to a defined 'subset' of L3 cache which may be overlapping
  * with other 'subsets'. The different subsets are identified by class of
  * service (CLOS) and each CLOS has a capacity bitmask (CBM).
  *
+ * Cache Monitoring Technology (CMT) supports monitoring of the last-level cache (LLC)
+ * for each running thread simultaneously.
+ *
  * Memory Bandwidth Allocation (MBA) provides indirect and approximate throttle
  * over memory bandwidth for the software. A user controls the resource by
  * indicating the percentage of maximum memory bandwidth or memory bandwidth
  * limit in MBps unit if MBA Software Controller is enabled.
  *
+ * Memory Bandwidth Monitoring (MBM) supports monitoring of memory bandwidth total and local
+ * for each running thread simultaneously.
+ *
  * More details about Intel RDT CAT and MBA can be found in the section 17.18
```

Comment:
"More details about Intel RDT hardware features can be found in the Section 17.18 and 17.19, Volume 3"